tests/integration_category_colors/CSS/Assets_Test.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Tribe\Events\Category_Colors\CSS;
+
+use TEC\Events\Category_Colors\CSS\Assets;
+use TEC\Events\Category_Colors\CSS\Generator;
+use Tribe\Tests\Traits\With_Uopz;
+use Codeception\TestCase\WPTestCase;
+
+class Assets_Test extends WPTestCase {
+    use With_Uopz;
+
+    /**
+     * @var Assets
+     */
+    protected $assets;
+
+    /**
+     * @var Generator
+     */
+    protected $generator;
+
+    /**
+     * @before
+     */
+    public function before() {
+        $this->generator = tribe( Generator::class );
+        $this->assets = tribe( Assets::class );
+        
+        // Reset WordPress styles and scripts
+        global $wp_styles, $wp_scripts;
+        $wp_styles = new \WP_Styles();
+        $wp_scripts = new \WP_Scripts();
```

Comment:
Just for your info:
If you add these via a `use` above you don't need to add the backslash here.